### ISSUE TITLE
Prediction

### ISSUE THREAD

**Issue by dasrakesh** [nish-nm/braTS2015-challenge#1]
Bro how to predict particular .mah file and how to show graphs for learning rate, loss function and accuracy rate?
